web-common/src/features/models/workspace/CreateDashboardButton.svelte

```diff
@@ -101,11 +103,17 @@
   }
 </script>
 
-<Tooltip alignment="right" distance={16} location="bottom">
-  <Button on:click={handleCreateDashboard} type="primary">
-    <ResponsiveButtonText {collapse}>Create Dashboard</ResponsiveButtonText>
-    <Explore size="14px" /></Button
+<Tooltip alignment="right" distance={8} location="bottom">
+  <Button
+    disabled={!timestampColumns?.length}
+    on:click={handleCreateDashboard}
+    type="primary"
   >
+    <IconSpaceFixer pullLeft pullRight={collapse}>
+      <Add />
+    </IconSpaceFixer>
+    <ResponsiveButtonText {collapse}>Create Dashboard</ResponsiveButtonText>
```

Comment:
There is no tooltip content for the Create Dashboard button. 
This is also present in main but probably best to handle in this PR.

![image](https://user-images.githubusercontent.com/4402679/213146895-8a5250cf-1987-48fc-a20e-3d86d2b4f44f.png)